ext/opentelemetry-ext-flask/tests/test_flask_integration.py

```diff
@@ -14,13 +14,12 @@
 
 import unittest
 
-from flask import Flask
-from werkzeug.test import Client
-from werkzeug.wrappers import BaseResponse
-
 import opentelemetry.ext.flask as otel_flask
+from flask import Flask
 from opentelemetry import trace as trace_api
 from opentelemetry.ext.testutil.wsgitestutil import WsgiTestBase
+from werkzeug.test import Client
+from werkzeug.wrappers import BaseResponse
```

Comment:
surprised to see these imports order needing to change

Reply:
I'm sorting through it, something to do with how isort resolves first party vs third-party imports.

Reply:
Try deleting your virtualenv. I had troubles with `isort` too and it seems that if there is a `site-packages/openetelemetry` directory (even an empty one), it will think of `opentelemetry` as a 3rd party. On the other hand, if you install it editable, there is only a `opentelemetry.egg-link` file, which `isort` doesn't find, so it falls back to thinking of `opentelemetry` as a first-party.

Reply:
For reference: https://github.com/timothycrosley/isort/issues/1104